browser/widevine/widevine_permission_request_browsertest.cc

```diff
@@ -256,13 +256,17 @@ IN_PROC_BROWSER_TEST_F(ScriptTriggerWidevinePermissionRequestBrowserTest,
   ASSERT_TRUE(ui_test_utils::NavigateToURL(browser(), url));
   EXPECT_FALSE(IsPermissionBubbleShown());
 
+  const std::string js_error =
+      "a JavaScript error: \"NotSupportedError: Unsupported keySystem or "
+      "supportedConfigurations.\"\n";
+
   const std::string drm_js =
       "var config = [{initDataTypes: ['cenc']}];"
       "navigator.requestMediaKeySystemAccess($1, config);";
   const std::string widevine_js = content::JsReplace(drm_js,
                                                      "com.widevine.alpha");
 
-  EXPECT_TRUE(content::ExecJs(active_contents(), widevine_js));
+  EXPECT_EQ(js_error, content::EvalJs(active_contents(), widevine_js).error);
```

Comment:
is this error also expected in Release builds with Widevine fully enabled?

Reply:
we do, b/c widevine is only dynamically downloaded after an explicit action with the bubble.

Reply:
yeah, we'll get this error till Widevine is downloaded and loaded.